core/src/test/java/org/mache/MongoDbForTestsPresent.java

```diff
@@ -0,0 +1,12 @@
+package org.mache;
+
+public class MongoDbForTestsPresent extends TestEnvironmentPortCheckIgnoreCondition
+{
```

Comment:
As a suggestion, these subclasses might read better when used in the @ignore annotation if they were named something like `MongoDbNotRunningLocally`
`@IgnoreIf(condition = MongoDbNotRunningLocally.class)`
This would help the intent, to ignore if there is no local mongo